server/py/services/api/crud/model_monitoring/deployment.py

```diff
@@ -1400,7 +1398,11 @@ async def create_model_endpoints(
             batch = model_endpoints_instructions[i : i + batchsize]
             coroutines.append(
                 MonitoringDeployment._create_model_endpoint_limited(
-                    semaphore, batch, project
+                    semaphore,
+                    batch,
+                    project,
+                    function_name=function.metadata.name,
+                    function_tag=function.metadata.tag or "latest",
```

Comment:
just to keep consistency
```suggestion
                    semaphore=semaphore,
                    model_endpoints_instructions=batch,
                    project=project,
                    function_name=function.metadata.name,
                    function_tag=function.metadata.tag or "latest",
```